edit-javaee/login.properties

```diff
@@ -0,0 +1,20 @@
+#Tue Jul 14 16:00:41 CEST 2015
+javax.persistence.jdbc.url=jdbc\:mysql\://localhost\:3306/items
+eclipselink.logging.session=false
+eclipselink.logging.level=FINE
+eclipselink.ddl-generation=drop-and-create-tables
+eclipselink.logging.timestamp=false
+eclipselink.application-location=C\:/Users/uporabnik/Desktop/edit/edit-2015/edit-javaee
```

Comment:
What is this file used for?
It also contains hard coded absolute path that is not portable.